extensions/VisualEditor/modules/ve-mw/init/targets/ve.init.mw.ViewPageTarget.js

```diff
@@ -986,6 +982,12 @@ ve.init.mw.ViewPageTarget.prototype.saveDocument = function ( saveDeferred ) {
 	}
 };
 
+ve.init.mw.ViewPageTarget.prototype.isSummaryMissing = function ( saveOptions ) {
+	return mw.user.options.get( 'forceeditsummary' ) &&
```

Comment:
This option comes as string value. In original code we're casting it to integer value.
I think we should keep this `+` to make it work correctly in all cases.